src/main/java/com/kodilla/ecommercee/domains/Group.java

```diff
@@ -0,0 +1,37 @@
+package com.kodilla.ecommercee.domains;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Getter
+@Setter
```

Comment:
Czy settery są konieczne?

Reply:
Wydaje mi się, że jeżeli adnotację są umieszczane przy polach klasy to settery są konieczne.

Reply:
jeżeli były by umieszczane przy getterach to wtedy byłyby konieczne settery, jeżeli są poprostu nad polem to nie potrzebujemy setterów

Reply:
Zgodnie z tym co napisałem :)

Reply:
ok, moj bład, zle przeczytałem Twoja wypowiedz ;D nie bylo tematu xD

Reply:
ok, do usunięcie @Setter